platform/ui/src/components/Header/Header.jsx

```diff
@@ -40,16 +45,32 @@ function Header({
         </div>
         <div className="flex items-center">{children}</div>
         <div className="flex items-center">
-          <IconButton
-            id={'options-settings-icon'}
-            variant="text"
-            color="inherit"
-            size="initial"
-            className="text-primary-active"
-            onClick={onClickSetting}
-          >
-            <Icon name="settings" />
-          </IconButton>
+          <Tooltip content="Copy Instance Link">
+            <IconButton
+              id={'options-clipboard-icon'}
```

Comment:
id is used in the IconButton constructor and as far as I can tell is used in the data-cy field when constructing the button:

https://github.com/new-lantern/nl-ohif/blob/feat/nl-dev/platform/ui/src/components/IconButton/IconButton.jsx#L118

data-cy (Cypress )is used mostly for more fine-tuned element selection so I think it'd be good to keep in for now in case in the future we want to do targeted selection on the settings or clipboard buttons

https://docs.cypress.io/guides/references/best-practices